bms_receiver/bms_receiver_test.py

```diff
@@ -0,0 +1,166 @@
+import unittest
+from io import StringIO
+from mock import patch
+import bms_statistics_calculator as calc
+import bms_input_handler as bms_input
+import bms_output_handler as out
+import bms_receiver
+
+
+class BMSReceiverTest(unittest.TestCase):
+    def setUp(self):
+        temp_list = []
+        soc_list = []
+        calc.saved_bms_readings = {'Temperature': temp_list, 'StateOfCharge': soc_list}
+        calc.bms_statistics = {'Temperature': {'min': 0, 'max': 0, 'sma': 0},
+                               'StateOfCharge': {'min': 0, 'max': 0, 'sma': 0}}
+
+    def tearDown(self):
+        temp_list = []
+        soc_list = []
+        calc.saved_bms_readings = {'Temperature': temp_list, 'StateOfCharge': soc_list}
+        calc.bms_statistics = {'Temperature': {'min': 0, 'max': 0, 'sma': 0},
+                               'StateOfCharge': {'min': 0, 'max': 0, 'sma': 0}}
+
+    bms_parameters_with_range = {'Temperature': {'min': 0, 'max': 45}, 'StateOfCharge': {'min': 20, 'max': 80}}
+
+    """
+    ******************************************************
+                    File: bms_input_handler
+    ******************************************************
+    """
+
+    def test_is_input_empty(self):
```

Comment:
test says input_empty but code is supplying input_readings? Hard to understand purpose of the test...
name your test functions with the inputs and expected output.